base/shell/cmd/config.h

```diff
@@ -42,7 +42,7 @@
 /* Define one of these to select the used locale. */
 /*  (date and time formats etc.) used in DATE, TIME, */
 /*  DIR, PROMPT etc. */
-#define LOCALE_WINDOWS   /* System locale */
+/* #define LOCALE_WINDOWS */   /* System locale */
```

Comment:
Can you elaborate on this change?

Reply:
This definition never used (not found even in commented code) but his name conflicting with group of defines from winnls.h

Seems like a rudiment

Reply:
Thanks